miniapps/navier/ortho_solver.hpp

```diff
@@ -27,8 +27,10 @@ namespace navier
  */
 class OrthoSolver : public Solver
 {
+private:
+   MPI_Comm mycomm;
 public:
-   OrthoSolver();
+   OrthoSolver(ParFiniteElementSpace *fes_);
```

Comment:
Why not just pass the communicator? I don't see `fes_` used other than getting its communicator

Reply:
hehe I was thinking about that.